exporter.py

```diff
@@ -70,21 +89,34 @@ def run_prometheus_server(port, collectors, *args):
         time.sleep(1)
 
 
-def initialize_collector_by_name(class_name):
+def initialize_collector_by_name(class_name, test=False):
     try:
-        class_module = importlib.import_module('collectors.%s' % (class_name))
-    except ModuleNotFoundError:
-        print('No Collector "%s" defined. Ignoring...' % (class_name))
+        class_module = importlib.import_module(f'collectors.{class_name}')
+    except ModuleNotFoundError as e:
+        if test:
+            print('No Collector "BogusCollector" defined. Ignoring...')
```

Comment:
Instead of specifying `test` shouldn't we expect the code to fail without giving it an explicit test option? Feels like it kind of kills the purpose.

Reply:
bump^